petclinic.yaml

```diff
@@ -11,16 +11,21 @@ tools:
     type: dockerImage
     image: maven:3.5.4-jdk-8
     entryPoint: sleep infinity
-    persistentVolumes:
+    volumes:
       - name: maven-repo
-        containerPath: /root/.m2
-    services:
+        mountPath: /root/.m2
+      - name: projects
```

Comment:
Agree with @l0rd it would be easier to not to write /projects and let that to the implementation to decide on which path it's coming)

Reply:
well, a user is able to specify any path he needs for projects folder to be mounted inside of his container.

Reply:
> User can specify all the mount volumes he might want. But if I mount /projects I expect that all ppl mounting projects source-tree will have the same data across the workspace and not get their own volume as well.

Che Volumes should mount the same data if volumes name is the same.
And also the user is able to mount volume where he needs it to be mount inside of a container.
Example:
```yaml
  - name: mvn-stack
    type: dockerImage
    image: maven:3.5.4-jdk-8
    entryPoint: sleep infinity
    volumes:
      - name: maven-repo
        mountPath: /root/.m2
  - name: gradle-stack
    type: dockerImage
    image: gradle:3.5.4-jdk-8
    entryPoint: sleep infinity
    volumes:
      - name: maven-repo
        mountPath: /home/user/.m2
```
`mvn-stack` and `gradle-stack` tools should have the same data but mount to different paths.
It is how I understand the format described by @l0rd. Maybe it is not related to Che Volumes (what would be confusing) at all.